tomcat-8.5/src/org/dogtagpki/tomcat/JSSUtil.java

```diff
@@ -19,38 +19,89 @@
 
 package org.dogtagpki.tomcat;
 
+import java.util.List;
+import java.util.Set;
+import java.util.HashSet;
+
 import javax.net.ssl.KeyManager;
 import javax.net.ssl.TrustManager;
 
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
 import org.apache.tomcat.util.net.SSLHostConfigCertificate;
 import org.apache.tomcat.util.net.jsse.JSSEKeyManager;
-import org.apache.tomcat.util.net.jsse.JSSEUtil;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.tomcat.util.net.SSLContext;
+import org.apache.tomcat.util.net.SSLUtil;
+import org.apache.tomcat.util.net.SSLUtilBase;
 
 import org.mozilla.jss.provider.javax.crypto.JSSKeyManager;
 import org.mozilla.jss.provider.javax.crypto.JSSTrustManager;
+import org.mozilla.jss.ssl.SSLCipher;
+import org.mozilla.jss.ssl.SSLVersion;
 
-public class JSSUtil extends JSSEUtil {
+public class JSSUtil extends SSLUtilBase {
+    public static Log logger = LogFactory.getLog(JSSUtil.class);
 
-    public static Logger logger = LoggerFactory.getLogger(JSSUtil.class);
```

Comment:
I had to replace this as that's what Tomcat expects to be returned from `getLog()`. ~shrugs~